### DISCUSSION TITLE
希望可以隐藏侧边停靠栏

### DISCUSSION THREAD

**Discussion by veizei-K** [the1812/Bilibili-Evolved#4660] [功能建议 / Ideas]
希望可以隐藏侧边停靠栏，直接从油猴插件的脚本那里点击进入。
<img width="59" alt="PixPin_2024-03-01_12-48-50" src="https://github.com/the1812/Bilibili-Evolved/assets/129288685/d68bd01b-d46c-4cac-a48a-0edce9a18916">
![image](https://github.com/the1812/Bilibili-Evolved/assets/129288685/39e5c8cd-f3eb-4b51-860d-e52f9838054a)
这样看着比较清爽

**Comment by Geeyun-JY3** [comment]
请善用搜索，认真阅读文档：https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features/features.md#自动隐藏侧栏
